{{cookiecutter.python_name}}/binder/environment.yml

```
@@ -12,7 +12,7 @@ channels:
 dependencies:
   # runtime dependencies
   - python >=3.8,<3.9.0a0
-  - jupyterlab >=3,<4.0.0a0
+  - jupyterlab >=4.0.0a29,<5
   # labextension build dependencies
   - nodejs >=14,<15
```

Should we upgrade nodejs (and maybe Python)?

```suggestion
  - nodejs >=16,<17
```

---

Thanks, sounds good to me.